lib/braintree_blue/api.rb

```diff
@@ -91,7 +91,8 @@ def search_payments(search_key, offset, limit, properties, context)
       end
 
       def add_payment_method(kb_account_id, kb_payment_method_id, payment_method_props, set_default, properties, context)
-       braintree_customer_id = BraintreeBluePaymentMethod.braintree_customer_id_from_kb_account_id(kb_account_id, context.tenant_id)
+        braintree_customer_id = payment_method_props.properties.find { |p| p.key == 'customer' }.try(:value) ||
```

Comment:
Nit: you could reuse the `find_value_from_properties` helper ([implementation](https://github.com/killbill/killbill-plugin-framework-ruby/blob/master/lib/killbill/helpers/properties_helper.rb#L5)), which should already be available in `api.rb`.